pkg/collector/process_test.go

```diff
@@ -64,11 +64,11 @@ func TestTCPCollectingProcess_ReceiveTemplateRecord(t *testing.T) {
 			messageReceived <- true
 		}
 	}()
-	time.Sleep(4 * time.Second)
+	isReceived := <-messageReceived
```

Comment:
There is only one message right? Can't we directly use that instead of creating one again? In addition, we can get away with extra go routine and directly use `go cp.Start()`.

I see that they same format used in rest of the tests.